### PR TITLE
S1 bt2

### PR DESCRIPTION
**Pull request by RachmannJoubert**
Populating the database with relevent values and images.

### CODE REVIEW

**Comment by MotNosniktaw** [comment]
output seems to match table structure

**Comment by MotNosniktaw** [comment]
table output matches expected. all good

**Comment by AlisonYoon** [comment]
Why don't you change canopy_level's data type to ENUM, like right now?

**Review by AlisonYoon — CHANGES_REQUESTED** [review]
change data type of canopy_level to ENUM

**Comment by AlisonYoon** [comment]
If you want all the fields to be mandatory, you probably want to change their default value to be "Not Null".

**Review by AlisonYoon — APPROVED** [review]

File: Project2_2019-08-05.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+LOCK TABLES `Plants` WRITE;
+/*!40000 ALTER TABLE `Plants` DISABLE KEYS */;
+
+INSERT INTO `Plants` (`id`, `common_name`, `latin_name`, `identification_image`, `canopy_level`, `life_span`)
+VALUES
+	(1,'American Persimmon','Diospyros virginiana','./images/american-persimmon.jpg','CANOPY LAYER','60'),
+	(2,'Nodding Saltbush','Einadia wutans','./images/nodding-saltbush.jpg','SHRUB LAYER','10'),
+	(3,'Chinese Chestnut','Castanea mollissima','./images/chinese-chestnut.jpg','CANOPY LAYER','150'),
+	(4,'papaya tree','carica papaya','./images/yellow-papaya-tree.png.webp','LOW-TREE LAYER','5'),
```
Comment (sjr-mayden) [inline Project2_2019-08-05.sql]: I'd take a look at the capitalisation for each of these entries to make sure it's consistent between entries and grammatically correct

File: Project2_2019-08-05.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	(5,'Black locust','Robinia pseudoacacia ','./images/black-locust.jpg.webp','CANOPY LAYER','75'),
```
Comment (sjr-mayden) [inline Project2_2019-08-05.sql]: What happens if you remove the .webp file extension? These look weird and if you can make them neat and tidy to start with that's cool

File: Project2_2019-08-05.sql [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+DROP TABLE IF EXISTS `Plants`;
+
+CREATE TABLE `Plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `common_name` varchar(255) NOT NULL DEFAULT '',
+  `latin_name` varchar(255) NOT NULL DEFAULT '',
+  `identification_image` varchar(255) NOT NULL DEFAULT '',
+  `canopy_level` enum('CANOPY LAYER','LOW-TREE LAYER','SHRUB LAYER','HERBACIOUS LAYER','GROUNDCOVER LAYER','RHIZOSPHERE LAYER','VERTICAL LAYER') NOT NULL DEFAULT 'CANOPY LAYER',
+  `life_span` varchar(255) NOT NULL DEFAULT '',
```
Comment (sjr-mayden) [inline Project2_2019-08-05.sql]: This isn't very locked down, anything could be entered here at the moment a numerical data type would probably work better

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: Project2_2019-08-05.sql (hunk 2 above)
Comment (RachmannJoubert) [reply Project2_2019-08-05.sql]: The images didn't work without the extensions so i just replaced them with new images that only have .jpg

**Pull request merged**